pkg/httpmock/stub.go

```diff
@@ -56,6 +56,26 @@ func GraphQL(q string) Matcher {
 	}
 }
 
+func EnterpriseGraphQL(q string) Matcher {
+	re := regexp.MustCompile(q)
+
+	return func(req *http.Request) bool {
+		if !strings.EqualFold(req.Method, "POST") {
+			return false
+		}
+		if req.URL.Path != "/api/graphql" {
```

Comment:
I can tell why you needed to add an extra matcher, but I'm not sure if it's worth a new method just to match an alternate API endpoint. Could you instead amend the `GraphQL` matcher to match on either `/graphql` or `/api/graphql` indiscriminately? Most assertions should not be concerned with whether the request was "Enterprise" or not; and our API routing is already tested elsewhere.

Reply:
that makes perfect sense, i'll do that.